src/web/app/pages-admin/admin-home-page/admin-home-page.component.ts

```diff
@@ -64,12 +58,18 @@ export class AdminHomePageComponent implements OnInit {
         invalidLines.push(instructorDetail);
         continue;
       }
-      this.instructorsConsolidated.push({
-        name: instructorDetailSplit[0],
-        email: instructorDetailSplit[1],
-        institution: instructorDetailSplit[2],
-        status: 'PENDING',
-        isCurrentlyBeingEdited: false,
+
+      const requestData: AccountCreateRequest = {
+        instructorEmail: instructorDetailSplit[1],
+        instructorName: instructorDetailSplit[0],
+        instructorInstitution: instructorDetailSplit[2],
+      };
+
+      this.accountService.createAccountRequest(requestData)
```

Comment:
This batch processing needs better error handling, this implementation essentially ignores any errors. I understand that this is for the first input form. Perhaps a better way to do this is push back to the invalid lines upon error and show an error toast to indicate that the creation of certain requests. Note that requests may fail for different reasons too (duplicate email, invalid email etc.)

Reply:
It now on error, pushes the line to `invalidLines`, rejoins `invalidLines` then shows an error toast.